course5/homework05/vector2d.h

```diff
@@ -0,0 +1,51 @@
+#ifndef VECTOR2D
+#define VECTOR2D
+#include <ostream>
+#include <cstring>
+
+namespace homework05
+{
+using namespace std::literals;
```

Comment:
You are not using any of the literals provided by the standard library, so this is not needed.